### PR TITLE
Revert changes to skip print for ISOTP_KERNEL_MODUL unit tests

### LINKED ISSUE #2551: Revert print of skip message
Reminder: This changes have to be reverted. The purpose of subprocess.call was to display this message in the output of UTScapy. sys.stderr is redirected, and will not fulfill this purpose.

_Originally posted by @polybassa in https://github.com/secdev/scapy/pull/2459/review_comment/create_

### PR DESCRIPTION
**Pull request by polybassa**
Fixes #2551

### CODE REVIEW

File: test/contrib/automotive/ecu_am.uts [hunk 1]
```diff
@@ -18,7 +18,8 @@ iface1 = "vcan1"
 ISOTP_KERNEL_MODULE_AVAILABLE = False
 def exit_if_no_isotp_module():
     if not ISOTP_KERNEL_MODULE_AVAILABLE:
-        sys.stderr.write("TEST SKIPPED: can-isotp not available" + os.linesep)
+        err = "TEST SKIPPED: can-isotp not available"
+        subprocess.call("printf \"%s\r\n\" > /dev/stderr" % err, shell=True)
```
Comment (gpotter2) [inline test/contrib/automotive/ecu_am.uts:22]: What's the point of this actually?

**Comment by polybassa** [comment]
We want to have this output on UTscapys output during the test. This is the only way to print from inside a *.uts file directly to UTscapys overview section.
-- 
Viele Grüße

Nils Weiß

**Comment by gpotter2** [comment]
What about `print("hey", file=sys.__stdout__)` ?

**Comment by polybassa** [comment]
If I remember correctly, sys.__stdout__ is replaced through UTscapy https://github.com/secdev/scapy/blob/551c1b76e007ea5f5fc88ef00f61600d072b4f2c/scapy/autorun.py#L131

So opening a new process was the simplest way.
-- 
Viele Grüße

Nils Weiß

**Comment by gpotter2** [comment]
No, `sys.stdout` is replaced but the `__stdout__` version is a backup that can't be replaced

**Comment by polybassa** [comment]
Ah ok. I'll try it out. Thanks 
-- 
Viele Grüße

Nils Weiß

**Comment by polybassa** [comment]
Works.. Thanks a lot.

**Comment by gpotter2** [comment]
You can rebase against master: i removed the failing tests as a hotfix

**Comment by polybassa** [comment]
I will, but at the moment, it looks like all test will pass

**Pull request merged**